notebooks/kmeans-no-import-change.ipynb

```diff
@@ -19,16 +19,23 @@
     "# KMeans - no import change"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Uncommenting and running the next cell will redirect all subsequent `pyspark.ml` imports (e.g. `pyspark.ml.clustering.Kmeans`) to GPU accelerated `spark_rapids_ml` counterparts if they are supported.  Comment out and restart the kernel to revert to `pyspark.ml` CPU imports.\n",
+    "\n",
+    "If you are running this notebook in local mode after starting `pyspark` with the `spark-rapids-ml` CLI `pyspark-rapids` or on EMR, Databricks, or Dataproc by following the respective READMEs and have selected the \\*no-import\\* init/bootstrap scripts the cell need not be run as the import was already run as part of the kernel launch.\n",
+    "Note that in these cases, there is no way to revert to CPU mode without creating a new (CPU) cluster or using the baseline `pyspark` command for local mode."
```

Comment:
Nit: would it be possible to rephrase the command to be simpler in one sentence like "If you are running this notebook with pyspark-rapids, there is no way to revert to GPU mode."

Reply:
Done